app.py

```diff
@@ -0,0 +1,22 @@
+from config import Config
+from logger import logger
+from flask import Flask
```

Comment:
using ,?

Reply:
Oh, I guess it is both ok.
https://stackoverflow.com/questions/15011367/python-module-import-single-line-vs-multi-line
There also have some argue about this. 
 From a stylistic perspective, one might be more preferable than the other.